demos/python_demos/multi_camera_multi_person_tracking/utils/ie_tools.py

```diff
@@ -0,0 +1,99 @@
+"""
+ Copyright (c) 2018 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import sys
+import os
+
+import glog as log
```

Comment:
We don't use `logging` module instead?

Reply:
glog doesn't require any setting up activities to print everything to the console therefore it's slightly more convenient. Replaced it to log to reduce the number of dependencies.